### PR TITLE
[lit] convert verify test as lit FileCheck test part1.5

### PR DESCRIPTION
**Pull request by python3kgae**
Add non-lib profile RUN line when possible.

### CODE REVIEW

**Review by llvm-beanz — COMMENTED** [review]
As @tex3d suggested yesterday in our meeting, if the `lib` run lines are remaining we should also add the appropriate `shader` attribute on the entry functions. That allows us to rely on these tests to be consistent between `lib` and the explicit target profile as we evolve the diagnostics to more correctly utilize the target profile.

**Comment by python3kgae** [comment]
> As @tex3d suggested yesterday in our meeting, if the `lib` run lines are remaining we should also add the appropriate `shader` attribute on the entry functions. That allows us to rely on these tests to be consistent between `lib` and the explicit target profile as we evolve the diagnostics to more correctly utilize the target profile.

Updated.

**Comment by bob80905** [comment]
Also curious, why add ps_6_0 when there was originally a lib_6_3? Shouldn't the new run line be ps_6_3?

**Comment by python3kgae** [comment]
> Also curious, why add ps_6_0 when there was originally a lib_6_3? Shouldn't the new run line be ps_6_3?

Want to use the lowest shader model.

**Review by llvm-beanz — COMMENTED** [review]
There were 50 files in #5904, and this change only fixes 15. Do the remaining tests not have valid entry functions?

File: tools/clang/test/SemaHLSL/attributes.hlsl [hunk 1]
```diff
@@ -696,6 +696,7 @@ void uniform_maxvertexcount2(triangle GSVertex v[3], inout TriangleStream<GSVert
 { stream.Append(v[0]); }
```
[the remark this answers is not in the document]
Comment (python3kgae) [reply tools/clang/test/SemaHLSL/attributes.hlsl:699]: Fixed.

**Comment by python3kgae** [comment]
> There were 50 files in #5904, and this change only fixes 15. Do the remaining tests not have valid entry functions?

There're some void main() without [numofthread].
Also one case with main but not have attribute on the parameter.
None of these are valid entry functions.

**Comment by llvm-beanz** [comment]
> There're some void main() without [numofthread]. Also one case with main but not have attribute on the parameter. None of these are valid entry functions.

Sounds to me like those are valid vertex shader entries.

**Comment by python3kgae** [comment]
> > There're some void main() without [numofthread]. Also one case with main but not have attribute on the parameter. None of these are valid entry functions.
> 
> Sounds to me like those are valid vertex shader entries.

Marked as vs.

**Review by tex3d — APPROVED** [review]
LGTM!

**Review by llvm-beanz — APPROVED** [review]

**Pull request merged**